number_of_game_towers: 4
LED_per_tower: 4
# TOWER POSITIONS:
tower_1: [1.15, -0.87] 
tower_2: [3.22, -2.10]
tower_3: [1.14, -5.03]
tower_4: [-0.81, -3.60]
